src/cascadia/LocalTests_TerminalApp/TestHostApp/TestHostApp.vcxproj

```diff
@@ -0,0 +1,161 @@
+           TerminalAppLib, and when makepri.exe is eventually called, it'll run
+           into a duplicate entry for all the xbfs defined in that project.
+         * If you only reference TerminalAppLib here, the AppxManifest.xml will
+           think that TerminalAppLib.lib is the hosting binary for all the
+           TerminalApp types, which is _bad_
+
+         We'll manually reference the correct dll below.
+     -->
+  </ItemGroup>
+
+  <PropertyGroup>
+    <!-- Some helper paths to find our test code -->
+    <_TestBinRoot>$(OpenConsoleDir)\bin\$(Platform)\$(Configuration)</_TestBinRoot>
+    <_CppWinrtBinRoot>$(OpenConsoleDir)\$(Platform)\$(Configuration)</_CppWinrtBinRoot>
+    <_CppWinrtBinRoot Condition="'$(Platform)'=='Win32'">$(OpenConsoleDir)\$(Configuration)</_CppWinrtBinRoot>
+    <_TAEFPlatformName>$(Platform)</_TAEFPlatformName>
+    <_TAEFPlatformName Condition="'$(Platform)'=='Win32'">x86</_TAEFPlatformName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.VisualStudio.TestPlatform.TestExecutor.WinRTCore">
+      <HintPath>$(OpenConsoleDir)\packages\Taef.Redist.Wlk.10.48.200103003-develop\lib\Microsoft.VisualStudio.TestPlatform.TestExecutor.WinRTCore.winmd</HintPath>
+      <IsWinMDFile>true</IsWinMDFile>
+
+      <!-- This path is _relative to the .winmd_ -->
+      <Implementation>..\build\Binaries\$(_TAEFPlatformName)\TE.AppxUnitTestClient.dll</Implementation>
+    </Reference>
+  </ItemGroup>
+
+
+  <ItemGroup>
+    <!-- Manually reference TerminalApp -->
+    <Reference Include="TerminalApp">
+      <HintPath>$(_CppWinrtBinRoot)\TerminalApp\TerminalApp.winmd</HintPath>
+      <IsWinMDFile>true</IsWinMDFile>
+      <Implementation>TerminalApp.dll</Implementation>
+    </Reference>
+  </ItemGroup>
+
+  <Import Project="$(OpenConsoleDir)\packages\Microsoft.UI.Xaml.2.3.191217003-prerelease\build\native\Microsoft.UI.Xaml.targets" Condition="Exists('$(OpenConsoleDir)\packages\Microsoft.UI.Xaml.2.3.191217003-prerelease\build\native\Microsoft.UI.Xaml.targets')" />
+
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+
+
+  <Target Name="AfterBuild">
+
+    <!-- Copy the AppxManifest.xml to another file, because when TAEF is
+      deploying the app, it'll delete the AppxManifest.xml file from this
+      directory when it tries to clean up after itself. -->
+    <Copy SourceFiles="$(TargetDir)\AppxManifest.xml" DestinationFiles="$(TargetDir)\TestHostAppXManifest.xml" />
+
+    <!-- Copy our test code from LocalTests_TerminalApp into this directory -->
+    <Copy SourceFiles="$(_TestBinRoot)\LocalTests_TerminalApp\TerminalApp.LocalTests.dll" DestinationFiles="$(TargetDir)\TerminalApp.LocalTests.dll" />
+
+    <!-- Copy some dlls which TerminalConnection is dependent upon that didn't
+      get rolled up into this  directory -->
+
+    <Copy SourceFiles="$(_CppWinrtBinRoot)\TerminalConnection\cpprest142_2_10d.dll" DestinationFiles="$(TargetDir)\cpprest142_2_10d.dll" />

```

Comment:
These should be getting rolled up through TerminalConnection, and I do not understand quite why they're not..

Reply:
WTUniversal doesn't need this weird stuff